config/initializers/money.rb

```diff
@@ -0,0 +1,111 @@
+require 'money-rails'
```

Comment:
Is there any way to defer loading all of this...that is, it will get loaded on every process boot, even if no one uses chargeback

Reply:
Yes, I was thinking I needed to do that too. I should be able to make that happen.